src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const declarations = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter(Boolean);
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':').map((part) => part.trim());
```

Comment:
[CODE KNOWLEDGE] - There is no need to use `map` to trim the parts after splitting the declaration by ':'. Since you're not modifying an existing array but creating a new one from the split result, you can simply trim the parts directly within the destructuring assignment.